src/firefly/plans/xafs_scan.py

```diff
@@ -121,6 +154,9 @@ def update_line_edit_value(self, line_edit, conversion_func):
     def update_wavenumber_energy(self):
         is_k_checked = self.k_space_checkbox.isChecked()
```

Comment:
Not a problem, but I think you could just accept *is_k_checked* as an argument to this slot since k_space_checkbox.stateChanged emits it's new state as an argument.

Reply:
solved